### PR TITLE
bots: Adjust Debian image creation script for Debian 9

### PR DESCRIPTION
**Pull request by martinpitt**
Debian 9 was released a few weeks ago and now is the new "stable". pcp does not exist there (just like formerly in testing), so drop it there as well to [unbreak the image build](https://fedorapeople.org/groups/cockpit/logs/image-refresh-7083-20170627-140404/log). This fixes issue #7083.

Now that debian-stable is Debian 9 (Stretch), we can drop a lot of skips and special cases that only applied to Debian 8 (Jessie).

 - [x] fix missing ed25519 host key (`test/verify/check-system-info TestSystemInfo.testBasic`, `check-multi-machine`)
 - [x] adjust naughty overrides (`check-storage-msdos`)
 - [x] add missing "acl" package (`test/verify/check-keys`)
 - [x] relax CPU type display in `check-machines`
 - [x] fix `check-connection TestConnection.testBasic` → missing `systemd-coredump`

### CODE REVIEW

**Comment by stefwalter** [comment]
Does this mean we start releasing to Debian 9 or both Debian 8 and 9?

**Comment by martinpitt** [comment]
My gut feeling would be to only release to Debian 9 from now on. The primary way for users to get this should be through stretch-backports (although getting it there is still blocked a few days by the new cockpit being stuck in the binary NEW queue). So we can either change cockpituous to move to Debian 9, or to stop having our own Debian archive altogether. The thing is, once you create an archive it's rather unfriendly to take it down while the target release is still being supported, so we only have the chance of cleanup when Debian does a new stable release.

Debian 8 will still get security updates for about 10 months, then it's end of life. I could also put newer cockpits into the official -backports(-sloppy) for Debian 8 (once the Debian 9 backport is settled), but I'm not sure if that's really worth the effort.

**Comment by stefwalter** [comment]
> My gut feeling would be to only release to Debian 9 from now on. 

Makes sense to me.

**Comment by martinpitt** [comment]
I triggered the other debian/ubuntu tests to make triple sure that they didn't get broken by dropping the Debian 8 special cases.

File: bots/naughty/debian-stable/5399-storaged-crash-poll-with-variant [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Traceback (most recent call last):
+  File "./verify/check-storage-lvm2", line 155, in testLvm
```
Comment (dperpeet) [inline bots/naughty/debian-stable/5399-storaged-crash-poll-with-variant]: if we add this as s new naughty entry, shouldn't we wildcard the line number?

File: bots/naughty/debian-stable/5673-storaged-crash-udev-get-syspath [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Traceback (most recent call last):
+  File "./verify/check-storage-mounting", line 65, in testMounting
```
Comment (dperpeet) [inline bots/naughty/debian-stable/5673-storaged-crash-udev-get-syspath]: if we add this as s new naughty entry, shouldn't we wildcard the line number?

File: bots/naughty/debian-stable/5399-storaged-crash-poll-with-variant (hunk 1 above)
Comment (martinpitt) [inline bots/naughty/debian-stable/5399-storaged-crash-poll-with-variant]: I just copied them from debian-testing. However, it seems that #5399 only happened once, in March, on debian-testing. And #5673 has basically never matched. Also, the tests succeed in our current logs, so I think we can just drop these files altogether.

**Comment by martinpitt** [comment]
I dropped these two overrides from debian-stable and -testing, and re-triggered both tests.

**Pull request merged**